package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "npm explore @daostack/arc -- npm install openzeppelin-solidity @daostack/infra --no-save && npm explore @daostack/infra -- npm install openzeppelin-solidity --no-save && npm explore @daostack/arc-hive -- npm install openzeppelin-solidity --no-save",
```

Comment:
Seems weird to install a package inside an installed package on an npm-script. Why is'nt this part of `devDeps`

Reply:
truffle-flattener won't work otherwise

Reply:
weird, I don't understand the details but at the very least we can move this into `postinstall` to avoid having to remember to invoke this step manually.

Reply:
verify.initialize and verify.build are already run in prepare-release, prior to running verify.